### PR TITLE
Replace SSL commands with `cert-attach`.

### PR DESCRIPTION
**Pull request by ejholmes**
This goes along with https://github.com/remind101/empire/pull/701. Removes the ssl commands and adds a `cert-attach` command.

### CODE REVIEW

File: certs.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	"github.com/remind101/emp/Godeps/_workspace/src/github.com/bgentry/heroku-go"
+)
+
+var cmdCertAttach = &Command{
+	Run:      runCertAttach,
+	Usage:    "cert-attach",
+	NeedsApp: true,
+	Category: "certs",
+	Short:    "attach a certificate to an app",
+	Long:     `Attach a certificate to an app`,
```
Comment (phobologic) [inline certs.go]: How about:

`Attaches an SSL key & certificate to an exposed web app's load balancer.`

File: ssl.go [hunk 2]
```diff
@@ -1,195 +0,0 @@
...
-	"fmt"
-	"io/ioutil"
-	"os"
-	"strings"
-	"time"
-
-	"github.com/remind101/emp/Godeps/_workspace/src/github.com/bgentry/heroku-go"
-)
-
-var cmdSSL = &Command{
```
Comment (phobologic) [inline ssl.go:13]: How about we provide output that says "Go check out the cert-attach command"

**Comment by phobologic** [comment]
:+1:

File: certs.go (hunk 1 above)
Comment (ejholmes) [inline certs.go]: Updated with example.

File: ssl.go (hunk 2 above)
Comment (ejholmes) [inline ssl.go:13]: Empire will return an error message if they try to use the ssl-\* commands with an old emp client. Do you think we should keep the ssl-\* commands in here?

Comment (phobologic) [inline ssl.go:13]: Hmm - I don't suppose there's a way to keep them, but not include them in help?  That way if someone tries to use them out of habit, we can at least respond with a `This is deprecated` error, but folks don't see them and attempt to use them just from the help.

File: certs.go (hunk 1 above)
Comment (phobologic) [inline certs.go]: Awesome

File: ssl.go (hunk 2 above)
Comment (ejholmes) [inline ssl.go:13]: I'll add in support for hidden commands and bring it back.

**Pull request merged**